src/plugin.js

```diff
@@ -86,10 +86,17 @@ const setupHlsHandlers = function(qualityLevels, hls) {
  * Sets up the event handlers for populating and updating the selectedIndex of the
  * QualityLevelList for Dash source.
  *
+ * @return {Function} cleanup function to restore beforeInitialize on Html5DashJS
  * @function setupDashHandlers
  */
 const setupDashHandlers = function() {
+  let oldBeforeInitialize = videojs.Html5DashJS.beforeInitialize;
```

Comment:
I agree it would be nice for beforeInitialize to be more of a registration function that you can register multiple callbacks to, but that would require changes in contrib-dash, so probably best to leave outside this scope